spec/models/cost_object_spec.rb

```diff
@@ -0,0 +1,10 @@
+require "rails_helper"
+
+describe CostObject do
+  let(:blank_values) { [nil, ''] }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.